docs/cmake_doc/options.rst

```diff
@@ -213,6 +213,23 @@ NRN_MPI_DYNAMIC:STRING=
 
   This option is ignored unless NRN_ENABLE_MPI_DYNAMIC=ON
 
+NRN_ENABLE_MUSIC:BOOL=OFF
+-------------------------
+  Enable MUSIC. MUlti SImulation Coordinator.
+
+  MUSIC must already be installed. See https://github.com/INCF/MUSIC.
+
+  MPI and Python must be enabled.
+
+  If MUSIC is installed but CMake cannot find its /path, augment the
+  semicolon separated list of paths ``-DCMAKE_PREFIX_PATH=...;/path;...``
+  CMake needs to find
+
+  .. code-block:: shell
+
+    /path/include/music.hh
+    /path/lib/libmusic.so.*
```

Comment:
I guess I don't understand. I thought above that the library is libmusic.a

Reply:
I think these were your changes @nrnhines 😆 I'll have to update the doc. Also I notice now that I've been searching for the `.a` file, now I'm wondering what do we want? A static or a dynamic link?

Reply:
Sorry for the confusion. The line 231 above is mine. And I was wondering if it should be changed to ```/path/lib/libmusic.a``` to reflect the ```FindMusic.cmake``` line ```find_library(MUSIC_LIBRARY libmusic.a PATHS ${MUSIC_LIBDIR})```. Now, am I correct? in saying that line 231 above should lose the trailing ```.*``` and  that line 7 in FindMusic.cmake should read
```
find_library(MUSIC_LIBRARY libmusic.so PATHS ${MUSIC_LIBDIR})
```

Reply:
If we want to link dynamically, yes - we should consistently refer to the `.so` file. Otherwise, if we prefer the static archive, then we put everywhere `.a`.
This will have implications on our released binaries/wheels, though. If we take the `.so` path and build releases with NRNMUSIC enabled then music must be installed on the target machine. If we go the `.a` route then we will effective package the MUSIC library along with our code. And the last option is to do the same trick as for MPI where we do the runtime loading if available. What makes more sense?
If we consider MUSIC support more advanced and not something available by default, then we build our wheels/binaries without MUSIC and the dynamic option becomes superfluous. In that case I would do the dynamic linking against the `.so`, which has to be found at compile time.